### PR TITLE
chore(view-pipeline): Namananand/ins 3640 ensure alignment between input and output fields on the

### PR DESCRIPTION
**Pull request by iamnamananand996**
Because

- ensure alignment between input and output fields on the

This commit

- ensure alignment between input and output fields on the

### CODE REVIEW

File: packages/toolkit/src/view/pipeline/view-pipeline/Head.tsx [hunk 1]
```diff
@@ -341,25 +370,12 @@ export const Head = ({
                       router.push("/login");
                     }}
                     className="!normal-case"
-                    variant="secondaryGrey"
+                    variant="secondaryColour"
```
Comment (EiffelFly) [inline packages/toolkit/src/view/pipeline/view-pipeline/Head.tsx:373]: @iamnamananand996 Could you help me point out where you get this design?

File: packages/toolkit/src/view/pipeline/view-pipeline/ViewPipeline.tsx [hunk 2]
```diff
@@ -81,8 +81,8 @@ export const ViewPipeline = () => {
         }}
         currentVersion={currentVersion}
       />
-      <div className="mx-auto flex flex-1 flex-row px-8">
-        <div className="flex h-full w-[718px] flex-col gap-y-6 py-10 pr-10">
+      <div className="flex flex-1 flex-row px-8">
+        <div className="w-3/5 gap-y-6 py-10 pr-10">
```
Comment (EiffelFly) [inline packages/toolkit/src/view/pipeline/view-pipeline/ViewPipeline.tsx:85]: @iamnamananand996 I think you misunderstand the demanded change, please take a look at how Hugging face maintain a fixed width on their page. Something like this 

https://huggingface.co/meta-llama/Meta-Llama-3-8B

**Review by EiffelFly — COMMENTED** [review]
@iamnamananand996 Please check these comment

File: packages/toolkit/src/view/pipeline/view-pipeline/Head.tsx (hunk 1 above)
Comment (iamnamananand996) [reply packages/toolkit/src/view/pipeline/view-pipeline/Head.tsx:373]: https://www.figma.com/file/eTh2TCwmqN4MxiHJ4vTnPH?type=design&node-id=914-55784&mode=design#751456561

File: packages/toolkit/src/view/pipeline/view-pipeline/Head.tsx [hunk 3]
```diff
@@ -321,6 +321,35 @@ export const Head = ({
...
+                        stroke="#1D2433"
+                        stroke-width="1.5"
+                        stroke-linecap="round"
+                        stroke-linejoin="round"
+                      />
+                    </svg>
```
[the remark this answers is not in the document]
Comment (iamnamananand996) [reply packages/toolkit/src/view/pipeline/view-pipeline/Head.tsx]: updated

File: packages/toolkit/src/view/pipeline/view-pipeline/ViewPipeline.tsx (hunk 2 above)
Comment (iamnamananand996) [reply packages/toolkit/src/view/pipeline/view-pipeline/ViewPipeline.tsx]: hugging face is using bellow.
```
md:col-span-7 
md:col-span-5
```

it is same as `w-7/12` and `w-5/12`, updated it now

**Review by EiffelFly — COMMENTED** [review]
@iamnamananand996 Please check the updated comment

**Comment by EiffelFly** [comment]
@iamnamananand996 There other issues

1. The pipelineDescription margin is missing

CleanShot 2024-04-29 at 14 46 11@2x

2. The pr-4 of the right section is redundant, cause the end result not aligned with the design

CleanShot 2024-04-29 at 14 49 17@2x

**Comment by EiffelFly** [comment]
@iamnamananand996 I can not keep helping you debug these style bugs. This should be caught by you. (The editor input is two short.)

CleanShot 2024-04-29 at 20 23 26@2x

Dani have a new design for this readme input, could you also implement the border to make it more clearer

image

File: packages/toolkit/src/view/pipeline/view-pipeline/Head.tsx [hunk 4]
```diff
@@ -130,242 +130,247 @@ export const Head = ({
...
+                      size="sm"
+                    >
+                      Log in to Clone
+                    </Button>
+                  )}
+                </React.Fragment>
+              ) : (
+                <HeaderControllerSkeleton />
+              )}
+            </div>
```
Comment (EiffelFly) [inline packages/toolkit/src/view/pipeline/view-pipeline/Head.tsx:373]: @iamnamananand996 The whole logic is not correct...

When user can_edit we display edit button and when user is login but can not edit we display clone button. Please double check the original impl

**Review by EiffelFly — COMMENTED** [review]
@iamnamananand996 There is also this issue

**Pull request merged**